docs/OV_Runtime_UG/ONNX_Support.md

```diff
@@ -0,0 +1,65 @@
+# ONNX Format Support {#ONNX_Format_Support}
+
+
+Since the 2020.4 release, OpenVINO™ has supported native usage of ONNX models. The `core.read_model()` method provides a uniform way to read OpenVINO IR and ONNX formats alike, which is the recommended approach to reading models. Example:
+
+@sphinxdirective
+.. tab:: C++
+
+   .. code-block:: cpp
+   
+      ov::Core core;
+      std::shared_ptr<ov::Model> model = core.read_model("model.xml")
+ 
+.. tab:: Python
+
+   .. code-block:: python
+
+      import openvino.runtime as ov
+      core = ov.Core()
+      model = core.read_model("model.xml")
+@endsphinxdirective
+
```

Comment:
I recommend adding here:

"While ONNX models are directly supported by OpenVINO™, it can be useful to convert them to IR format to take advantage of advanced OpenVINO optimization tools and features. For information on how to convert an ONNX model to IR format, see the [Converting an ONNX Model](../MO_DG/prepare_model/convert_model/Convert_Model_From_ONNX.md) page."

Reply:
done :)